examples/openid-connect/README.md

```diff
@@ -0,0 +1,70 @@
+# OpenID Connect Example
+
+This repo contains a sample app for implementing Sign in with Slack (OpenID Connect compatible). Checkout `app.js`. The code includes a few different options which have been commented out. As you play around with the app, you can uncomment some of these options to get a deeper understanding of how to use this library.
+
+Local development requires a public URL where Slack can send requests. In this guide, we'll be using [`ngrok`](https://ngrok.com/download). Checkout [this guide](https://api.slack.com/tutorials/tunneling-with-ngrok) for setting it up.
+
+Before we get started, make sure you have a development workspace where you have permissions to install apps. If you don’t have one setup, go ahead and [create one](https://slack.com/create). You also need to [create a new app](https://api.slack.com/apps?new_app=1) with the following App Manifest if you haven’t already.
+
+```yaml
+_metadata:
+  major_version: 1
+  minor_version: 1
+display_information:
+  name: my-openid-connect-app
+oauth_config:
+  redirect_urls:
+    # TODO: Replace the URL with our own one
+    - https://your-own-domain/slack/oauth_redirect
+  scopes:
+    user:
+      - openid
+      - email
+      - profile
+```
+
+## Install Dependencies
+
+```
+npm install
+```
+
+## Setup Environment Variables
+
+This app requires you setup a few environment variables. You can get these values by navigating to your app's [**BASIC INFORMATION** Page](https://api.slack.com/apps).
```

Comment:
```suggestion
This app requires you to setup a few environment variables. You can get these values by navigating to your app's [**BASIC INFORMATION** Page](https://api.slack.com/apps).
```

Reply:
Thanks. This sentence is taken from https://github.com/slackapi/node-slack-sdk/blob/main/examples/oauth-v2/README.md - we may want to update the original one too

Reply:
Sounds good, I fixed it in https://github.com/slackapi/node-slack-sdk/commit/74149a50efa0d0219670115ef1a2e3ac7ccbcd04